lib/board.rb

```diff
@@ -0,0 +1,57 @@
+class Board
+
+    attr_accessor :cells
+
+    def initialize
+      @cells = Array.new(9, " ")
+    end
+
+    def reset!
+      @cells.clear
+      @cells = Array.new(9, " ")
+    end
+
+    def first_display
+      puts " #{1} | #{2} | #{3} "
+      puts "-----------"
+      puts " #{4} | #{5} | #{6} "
+      puts "-----------"
+      puts " #{7} | #{8} | #{9} "
+    end
+
+    def display
+      puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+      puts "-----------"
+      puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+      puts "-----------"
+      puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    end
+
+    def position(board_choice)
+      choice = @cells[board_choice.to_i - 1]
```

Comment:
setting this as a variable is unneeded because a ruby method already returns the value of `@cells[board_choice.to_i - 1]` implicitly